### PR TITLE
Listview: reorder items

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1326
[Problem] Controllers - Reorder
[Solution]
 - add reorder feature to listview

[Demo]
reorder-1

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/list/reorder-list.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	}
+
+	// Callback for page before show
+	function pageBeforeShow() {
+		listviewWidget = tau.widget.Listview(listElement),
+		tau.event.on(dragButton, "click", dragElements);
+	}
+
+	//Callback for page hide
+	function pageHide() {
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/list/reorder-list.js:22]: Need to destroy the listviewWidget.

File: src/css/profile/mobile/common/button.less [hunk 2]
```diff
@@ -675,3 +625,52 @@ a.ui-btn {
 
 	}
 }
+
+// Common classes for icons, not only for buttons
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/button.less]: How about creating the common.less or icons.less?

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/button.less]: I thought the same :) ... I'll move this to icons.less

File: examples/mobile/UIComponents/components/list/reorder-list.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/list/reorder-list.js:22]: Done

File: src/css/profile/mobile/common/button.less (hunk 2 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/button.less]: Done.

File: examples/mobile/UIComponents/components/controllers/index.html [hunk 3]
```diff
@@ -87,6 +87,11 @@ <h1>
 						Chips
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="../list/reorder-list.html">
+						Reorder Button
```
Comment (HunseopJeong) [inline examples/mobile/UIComponents/components/controllers/index.html:92]: Is the Reorder Button same with Reorder List?

Comment (TomaszLukawskiSam) [reply examples/mobile/UIComponents/components/controllers/index.html:92]: Yes. It is the similar like case with expand button.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**